app/src/main/java/net/gsantner/markor/activity/DocumentActivity.java

```diff
@@ -339,10 +340,13 @@ public void onBackPressed() {
         }
     }
 
-    public GsFragmentBase showFragment(GsFragmentBase fragment) {
+    @Override
+    public boolean onKeyDown(int keyCode, KeyEvent event) {
```

Comment:
Just move this into base class IMO.

onKeyDown -> pass to currentVisibleFragment() if it is not null

i.e instead of DocumentActivity -> BaseActivitiy -> DocumentEditFragment
just
BaseActivity (receive and forward) -> BaseFragment function -> Override in DocumentEditFragment

Reply:
But in `BaseActivity`, there is no method `getCurrentVisibleFragment()` or `getPosFragment(getCurrentPos())`, we cannot get the fragment we need.